bin/bundler-search

```diff
@@ -12,4 +12,5 @@
 #  2. Which gem names to search (defaults to all gems)
 
 pattern="$1"; shift
+# shellcheck disable=SC2046
```

Comment:
Shellcheck dies if you run it against multiple files and there's a failure earlier on.

I'm working on a test suite that's going to run our dotfile aliases, programs, and functions on different platforms to test, and was planning for shellcheck to be the linting portion of this work. Maybe we should look at doing shellcheck in Hound?